client/index.html

```diff
@@ -106,12 +106,12 @@ <h1 class="mdc-typography--headline">Import attendees</h1>
 							
 							<!-- Default header values match up with HackGTeeny survey headers -->
 							<div class="mdc-textfield" data-mdc-auto-init="MDCTextfield">
-								<input class="mdc-textfield__input" type="text" id="name-header" value="name">
-								<label class="mdc-textfield__label mdc-textfield__label--float-above" for="name-header">CSV name header</label>
+								<input class="mdc-textfield__input" type="text" id="name-header">
```

Comment:
This needs to have placeholders, see
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-placeholder